source/common/quic/envoy_quic_client_connection.cc

```diff
@@ -12,6 +12,10 @@
 namespace Envoy {
 namespace Quic {
 
+namespace {
+constexpr uint8_t kEcnMask = 0x03;
```

Comment:
Can we share this mask so we don't need it in two places? Maybe we could add `QuicEcnCodepoint getQuicEcnCodepointFromTosByte()` method which is called from both places? This would avoid the need to cast twice, too.